source/expressions.tex

```diff
@@ -1636,18 +1624,43 @@
 The closure type for a \grammarterm{lambda-expression} has a public
 inline function call operator (for a non-generic lambda) or
 function call operator template (for a generic lambda)\iref{over.call}
-whose parameters and return type
-are described by the \grammarterm{lambda-expression}'s
-\grammarterm{parameter-declaration-clause} and \grammarterm{trailing-return-type}
-respectively, and whose
-\grammarterm{template-parameter-list} consists of
-the specified \grammarterm{template-parameter-list}, if any.
-The \grammarterm{requires-clause} of the function call operator template
+whose:
+\begin{itemize}
+\item \grammarterm{parameter-declaration-clause} is the \grammarterm{lambda-declarator}'s
+\grammarterm{parameter-declaration-clause},
+
+\item return type is \tcode{auto},
```

Comment:
I think that this isn't the return type, so much as it is the type in the *decl-specifier-seq*. (The *declared* return type is determined by the *trailing-return-type* if any, and the *actual* return type is what we deduce. Per [dcl.spec.auto.general]p3, the `auto` is just something that can appear in the *decl-specifier-seq*; it's not really a return type at all.) So:

```suggestion
\item \grammarterm{decl-specifier-seq} is the \grammarterm{lambda-declarator}'s \grammarterm{decl-specifier-seq} followed by \keyword{auto}, except that \keyword{constexpr} is also included if [...]
```

Also... should we say something about the *attribute-specifier-seq*?